src/components/HomeSearchBox/index.tsx

```diff
@@ -6,9 +6,29 @@ const HomeSearchBox = () => {
   const { t } = useTranslation('home')
   const router = useRouter()
   const [searchValue, setSearchValue] = useState<string>('')
+  const [active, setActive] = useState(0)
+  const [filtered, setFiltered] = useState<string[]>([])
+  const [isShow, setIsShow] = useState(false)
 
```

Comment:
Search suggestions should disappear when user clicks outside.

Suggested solution:
```
  useEffect(() => {
    const handleMouseClick = (event: MouseEvent) => {
      setIsSuggestionShown(false)
    }

    document.addEventListener('mousedown', handleMouseClick)

    return () => {
      document.removeEventListener('mousedown', handleMouseClick)
    }
  }, [])
```